newsfragments/3014.bug.rst

```diff
@@ -0,0 +1,10 @@
+Resolve function_abi AFTER condition is initialized 
+
+Problem
```

Comment:
This might be a bit verbose for release notes 😅 . Just ensure that the PR/Issue number has this "problem" information, then only L1/L10 is needed (i.e. the solution)